test/Rln.t.sol

```diff
@@ -51,26 +53,26 @@ contract RlnTest is Test {
     }
 
     function test__InvalidRegistration__InsufficientDeposit(uint256 idCommitment) public {
+        vm.assume(rln.isValidCommitment(idCommitment));
         uint256 badDepositAmount = MEMBERSHIP_DEPOSIT - 1;
         vm.expectRevert(abi.encodeWithSelector(InsufficientDeposit.selector, MEMBERSHIP_DEPOSIT, badDepositAmount));
         rln.register{value: badDepositAmount}(idCommitment);
     }
 
-    function test__InvalidRegistration__FullSet(uint256 idCommitmentSeed) public {
-        vm.assume(idCommitmentSeed < 2 ** 255 - SET_SIZE);
+    function test__InvalidRegistration__FullSet() public {
```

Comment:
Just for me to understand: why did we move away from a fuzz test here? Has anything changed with regards to the seed that we can't/aren't relying on it anymore?

Reply:
technically we are not testing any logic here, we're just checking if an error is thrown based on the size of the tree - thats why we dont need to fuzz id commitments here